tensorflow/lite/toco/import_tensorflow_test.cc

```diff
@@ -385,6 +384,127 @@ std::vector<std::pair<tensorflow::DataType, ArrayDataType>> UnaryTestTypes() {
           {DT_INT64, ArrayDataType::kInt64}};
 }
 
+class TensorContentTest : public ::testing::Test {
+ public:
+  template <ArrayDataType T>
+  std::vector<DataType<T>> ImportAndGetData(const NodeDef& node) {
+    Model model;
+    auto status = ImportNode(node, &model);
+    CHECK(status.ok()) << status.error_message();
+    const auto& array = model.GetArray("Node1");
+    return array.GetBuffer<T>().data;
+  }
+  template <class T>
+  void NodeWithTensorContent(std::initializer_list<int64_t> shape,
+                             tensorflow::DataType dtype, int64_t num_elements,
+                             NodeDef* node) {
+    node->set_op("Const");
+    node->set_name("Node1");
+
+    // An attribute describing the type of this const node.
+    AttrValue dtype_attr;
+    SetAttrValue(dtype, &dtype_attr);
+    (*node->mutable_attr())["dtype"] = dtype_attr;
+
+    std::unique_ptr<T[]> allocated_content(new T[num_elements]);
```

Comment:
is it possible to use make_unique here rather than new?

Reply:
Yes updated the code as per your requirement